common/web/types/src/kpj/kpj-file-reader.ts

```diff
@@ -1,4 +1,4 @@
-import * as xml2js from 'xml2js';
+import * as xml2js from '../deps/xml2js/xml2js.js';
```

Comment:
Possibly worth considering:  [subpath imports](https://nodejs.org/api/packages.html#subpath-imports).

In the `common/web/types/package.json`, you could define a package-internal import (say, `#xml2js`) and not have to worry about potential variations in the relative pathing.